From 8f91696432cf597639154fcaa1bf4ec3c7af2d27 Mon Sep 17 00:00:00 2001
From: Kevin Mihelich <kevin@archlinuxarm.org>
Date: Sun, 23 Feb 2020 14:56:53 -0700
Subject: [PATCH] comment time size check

busybox isn't ready for musl's 64-bit time changes on 32-bit architectures.
---
 runit/runsv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/runit/runsv.c b/runit/runsv.c
index ccc762d78..fdb005c73 100644
--- a/runit/runsv.c
+++ b/runit/runsv.c
@@ -61,7 +61,7 @@ static void gettimeofday_ns(struct timespec *ts)
 static void gettimeofday_ns(struct timespec *ts)
 {
 	BUILD_BUG_ON(sizeof(struct timeval) != sizeof(struct timespec));
-	BUILD_BUG_ON(sizeof(((struct timeval*)ts)->tv_usec) != sizeof(ts->tv_nsec));
+	//BUILD_BUG_ON(sizeof(((struct timeval*)ts)->tv_usec) != sizeof(ts->tv_nsec));
 	/* Cheat */
 	gettimeofday((void*)ts, NULL);
 	ts->tv_nsec *= 1000;
-- 
2.24.1

